bool waypoint_controlling # false for another program controlling
---
bool success